src/libraries/Common/src/System/IO/FileSystem.Attributes.Windows.cs

```diff
@@ -29,6 +29,17 @@ private static bool DirectoryExists(string? path, out int lastError)
                 ((data.dwFileAttributes & Interop.Kernel32.FileAttributes.FILE_ATTRIBUTE_DIRECTORY) != 0);
         }
 
+        private static bool DirectoryExistsSlim(string? path, out int lastError)
+        {
+            Interop.Kernel32.WIN32_FILE_ATTRIBUTE_DATA data = default;
+            lastError = FillAttributeInfoSlim(path, ref data, returnErrorOnNotFound: true);
+
+            return
+                (lastError == 0) &&
+                (data.dwFileAttributes != -1) &&
+                ((data.dwFileAttributes & Interop.Kernel32.FileAttributes.FILE_ATTRIBUTE_DIRECTORY) != 0);
+        }
```

Comment:
Adding these "Slim" functions seems to be an attempt to improve throughput, but the perf results shared don't really show a throughput improvement.  Seems like these changes should be reverted?